SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -6,10 +6,16 @@
 #include "SpatialConstants.h"
 
 #include "CoreMinimal.h"
+#include "Schema/Interest.h"
```

Comment:
move up to GDK includes pls

Reply:
why is this include here if you're forward-declaring?

Reply:
it doesn't build otherwise, but this might be pebcak more than anything else as I don't really know why - I thought because it specifies a function that returns query constraint that it would be required?

Reply:
it's here because I'm not actually forward declaring

Reply:
Because we are explicitly using the type rather than a reference or pointer to it

Reply:
(I think)